src/Mapper/Mapper.test.ts

```diff
@@ -111,6 +117,21 @@ describe("Mapper", () => {
         fakeModelId: modelId,
       };
 
+      relatedFieldTransformFunction = spy();
+      fakeRelatedFieldType = {
+        transform: relatedFieldTransformFunction, 
+      };
+
+      /* serialize: boolean;
```

Comment:
This comment can be removed.

Reply:
As can `relatedFieldTransformFunction` above and in the `let` statements

Reply:
Thanks. I removed the comment and the relatedFieldTransformFunction